host_core/native/hostcore_wasmcloud_native/src/par.rs

```diff
@@ -27,25 +27,30 @@ pub(crate) fn get_vendor(par: &ProviderArchive) -> Result<String, Error> {
 
 pub(crate) fn extract_claims(par: &ProviderArchive) -> Result<Claims, Error> {
     match par.claims() {
-        Some(c) => {
-            let metadata = c.metadata.unwrap_or_default();
-            let revision = crate::revision_or_iat(metadata.rev, c.issued_at);
-            Ok(crate::Claims {
-                issuer: c.issuer,
-                public_key: c.subject,
-                revision,
-                tags: None,
-                version: metadata.ver,
-                name: metadata.name,
-                expires_human: stamp_to_human(c.expires).unwrap_or("never".to_string()),
-                not_before_human: stamp_to_human(c.not_before).unwrap_or("immediately".to_string()),
-                ..Default::default()
-            })
-        }
+        Some(c) => Ok(convert_provider_claims(c)),
         None => Err(Error::Term(Box::new("No claims found in provider archive"))),
     }
 }
 
+/// Convert provider claims to our crate claims type
+pub fn convert_provider_claims(
```

Comment:
a) should this be public?
b) would this be more appropriate inside an `impl Into<crate::Claims> for wascap::jwt:Claims....` ?

Reply:
Oh sorry, this was left over from me importing the crate into my original bindle only crate. However, I think this is definitely better as a `From` impl, so I'll go add that